src/main/java/org/opensearch/neuralsearch/settings/NeuralSearchSettings.java

```diff
@@ -47,4 +47,13 @@ public final class NeuralSearchSettings {
         Setting.Property.IndexScope,
         Setting.Property.Dynamic
     );
+
+    public static final Setting<Integer> HYBRID_COLLAPSE_DOCS_PER_GROUP = Setting.intSetting(
+        "index.neural_search.hybrid_collapse_docs_per_group",
+        10,
```

Comment:
I believe that is default value for this setting, is this based on some experimental data? does it give good balance between accuracy and performance?